app/views/products/index.html.erb

```diff
@@ -17,7 +17,7 @@
         <% end %>
         <td><%= product.name %></td>
         <td><%= product.description %></td>
-        <td><%= number_to_currency product.unit_price %></td>
+        <td><%= number_to_currency product&.unit_price_in_cents %></td>
```

Comment:
Regardless, "Working" does not mean correct, we cannot have cents here as it grossly misrepresents the price of the product. Nobody going to by a $200 muffin. It does like like `Product#unit_price` needs a guard clause to return nil if no `unit_price_in_cents` is present.